examples/resnet/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
do we want the tests to be a package? If so we should make sure to do this for all test subdirs.

I kinda don't want it to be a package unless there's code sharing across test files so that we have to explicitly opt into importing stuff from test files. Just part of a general defensive policy wrt enforcing modularity.